src/components/Nav.js

```diff
@@ -10,13 +10,19 @@ class Nav extends Component {
         { className: 'clearfix' },
         dom.ul(
           { className: 'nav nav-pills pull-right' },
-          dom.li(null, Link({ to: '/orders', className: 'nav-orders-link' }, dom.span(null, 'Orders'))),
-          dom.li(null, Link({ to: '/chef', className: 'nav-chef-link' }, dom.span(null, 'Chef'))),
-          dom.li(null, Link({ to: '/bartender', className: 'nav-bartender-link' }, dom.span(null, 'Bartender')))
+          this.context.location.pathname.indexOf('/orders') > -1 ?
+            dom.li(
+              null, Link({ to: '/orders', className: 'nav-orders-link' }, dom.span(null, 'Orders'))
+          ) : null,
+          dom.li(null, Link({ to: '/', className: 'nav-home-link' }, dom.span(null, 'Home')))
         )
       )
     );
   }
 }
 
+Nav.contextTypes = {
+  location: PropTypes.object
```

Comment:
On line 13 - this.context.location. This is gives the child component access to the context passed down from the parent.